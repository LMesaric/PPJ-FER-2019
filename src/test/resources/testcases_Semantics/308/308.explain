semantic: not declared
note: 'print' on line 6
