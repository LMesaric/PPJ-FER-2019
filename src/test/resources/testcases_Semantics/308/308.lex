KR_INT 1 int
IDN 1 main
L_ZAGRADA 1 (
KR_VOID 1 void
D_ZAGRADA 1 )
L_VIT_ZAGRADA 1 {
KR_INT 2 int
IDN 2 x
OP_PRIDRUZI 2 =
BROJ 2 3
TOCKAZAREZ 2 ;
KR_INT 3 int
IDN 3 y
OP_PRIDRUZI 3 =
BROJ 3 2
TOCKAZAREZ 3 ;
KR_INT 4 int
IDN 4 z
TOCKAZAREZ 4 ;
IDN 5 z
OP_PRIDRUZI 5 =
IDN 5 x
PLUS 5 +
IDN 5 y
TOCKAZAREZ 5 ;
IDN 6 print
L_ZAGRADA 6 (
NIZ_ZNAKOVA 6 "z = %d/n"
ZAREZ 6 ,
IDN 6 z
D_ZAGRADA 6 )
TOCKAZAREZ 6 ;
KR_RETURN 7 return
BROJ 7 0
TOCKAZAREZ 7 ;
D_VIT_ZAGRADA 8 }
