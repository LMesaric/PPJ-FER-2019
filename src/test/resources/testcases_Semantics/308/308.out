<primarni_izraz> ::= IDN(6,print)
